instrumentation/jsonrpc4j-1.6/library/build.gradle.kts

```
@@ -0,0 +1,14 @@
+plugins {
+  id("otel.library-instrumentation")
+}
+
+val jsonrpcVersion = "1.6"
+val jacksonVersion = "2.13.3"
+
+dependencies {
+  implementation("com.github.briandilley.jsonrpc4j:jsonrpc4j:$jsonrpcVersion")
```

```suggestion
  library("com.github.briandilley.jsonrpc4j:jsonrpc4j:$jsonrpcVersion")
```

Not sure should we also change jackson dependency below